src/utils/upload.js

```diff
@@ -14,10 +14,15 @@ export default class Upload {
    * @param opts.progress {Function} A progress callback for the upload. It will receive an Object
    * argument with either ``"indeterminate": true``, or numeric ``current`` and ``total`` fields.
    * @param opts.params {Object} Additional parameters to pass on the upload init request.
+   * @param opts.chunkLen {Number} Chunk size for sending the file (integer number of bytes).
    */
-  constructor($rest, file, parent, { progress = () => null, params = {} } = {}) {
+  constructor($rest, file, parent, {
+    progress = () => null,
+    params = {},
+    chunkLen = UPLOAD_CHUNK_SIZE,
```

Comment:
This was added simply so that we could unit test this using very small blobs. Maybe there will be some actual utility for it as well.